routers/web/repo/issue.go

```diff
@@ -136,6 +137,27 @@ func MustAllowPulls(ctx *context.Context) {
 	}
 }
 
+func dependenciesToHTML(ctx *context.Context, dependencies []*issues_model.DependencyInfo) template.HTML {
+	if len(dependencies) == 0 {
+		return ""
+	}
+
+	htmlCode := "<span>"
+
+	for index, dependency := range dependencies {
+		if index != 0 {
+			htmlCode += `<span>,</span>`
+		}
+		anchorClasses := "gt-ml-2 ref-issue"
+		if dependency.Issue.IsClosed {
+			anchorClasses += " gt-line-through"
+		}
+		htmlCode += fmt.Sprintf(`<a href="%s" class="%s">#%d</a>`, dependency.Issue.Link(), anchorClasses, dependency.Issue.Index)
```

Comment:
If nobody knows how to deal with RTL for this case, I would still suggest to render the HTML by Golang template, but not by backend Go code.